<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Post3dWindowNodeVectorArrowGroupDataItem_SettingEditWidget</class>
 <widget class="QWidget" name="Post3dWindowNodeVectorArrowGroupDataItem_SettingEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="basicTab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QSplitter" name="basicSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="ArrowsLengthSettingEditWidget" name="lengthSettingWidget" native="true"/>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>17</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QGroupBox" name="legendTextGroupBox">
          <property name="title">
           <string>Length Legend</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="ArrowsLegendSettingEditWidget" name="legendWidget" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="faceTab">
      <attribute name="title">
       <string>Faces</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0">
       <item>
        <widget class="QSplitter" name="facesSplitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QListWidget" name="faceListWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>124</width>
             <height>393</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="Post3dWindowFaceSettingEditWidget" name="faceSettingWidget" native="true"/>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="Structured3dFilteringSettingEditWidget" name="filteringWidget" native="true"/>
              </item>
              <item>
               <widget class="ArrowsColorSettingEditSmallWidget" name="colorSettingWidget" native="true"/>
              </item>
              <item>
               <widget class="ArrowsShapeSettingEditWidget" name="shapeWidget" native="true"/>
              </item>
              <item>
               <spacer name="verticalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="faceAddButton">
           <property name="text">
            <string>&amp;Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="faceRemoveButton">
           <property name="text">
            <string>&amp;Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Post3dWindowFaceSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>post/post3d/post3dwindowfacesettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowsLegendSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/arrows/arrowslegendsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowsLengthSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/arrows/arrowslengthsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowsShapeSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/arrows/arrowsshapesettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArrowsColorSettingEditSmallWidget</class>
   <extends>QWidget</extends>
   <header>guicore/arrows/arrowscolorsettingeditsmallwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Structured3dFilteringSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/filter/structured3dfilteringsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
